src/common/droid-util.c

```diff
@@ -1906,48 +1910,105 @@ static pa_droid_stream *droid_stream_new(pa_droid_hw_module *module) {
     return s;
 }
 
+static bool stream_config_fill(audio_devices_t devices,
+                               pa_sample_spec *sample_spec,
+                               pa_channel_map *channel_map,
+                               struct audio_config *config) {
+    audio_format_t hal_audio_format = 0;
+    audio_channel_mask_t hal_channel_mask = 0;
+    bool voicecall_record = false;
+
+    pa_assert(sample_spec);
+    pa_assert(channel_map);
+    pa_assert(config);
+
+#if AUDIO_API_VERSION_MAJ >= 2
+    if ((devices & ~AUDIO_DEVICE_BIT_IN) & AUDIO_DEVICE_IN_VOICE_CALL)
+#else
+    if (devices & AUDIO_DEVICE_IN_VOICE_CALL)
+#endif
+        voicecall_record = true;
```

Comment:
Thinking out loud: would it be better from readability pov to duplicate the assignment within the #if #else blocks?